lib/internal/source_map/prepare_stack_trace.js

```diff
@@ -88,15 +91,26 @@ const prepareStackTrace = (globalThis, error, trace) => {
 // Places a snippet of code from where the exception was originally thrown
 // above the stack trace. This logic is modeled after GetErrorSource in
 // node_errors.cc.
-function getErrorSource(firstSource, firstLine, firstColumn) {
+function getErrorSource(payload, originalSource, firstLine, firstColumn) {
   let exceptionLine = '';
+  const originalSourceNoScheme = originalSource.replace(/^file:\/\//, '');
```

Comment:
Better use `url.fileUrlToPath` util function to handle escaping correctly, and windows paths